packages/portugal/README.md

```diff
@@ -0,0 +1,48 @@
+# pt Portugal
+
+[![License: CC BY-SA 4.0](https://img.shields.io/badge/License-CC%20BY--SA%204.0-blue.svg)](https://creativecommons.org/licenses/by-sa/4.0/)
+
+Contains all the districts  of Portugal:
+* Azores
+* Aveiro
+* Beja
+* Braga
+* Bragança
+* Castelo Branco
+* Coimbra
+* Évora
+* Faro
+* Guarda
+* Leiria
+* Lisbon
+* Madeira
+* Portalegre
+* Porto
+* Santarém
+* Setúbal
+* Viana do Castelo 
+* Vila Real
+* Viseu
+
+
+## Installation
+
+### npm
+
+`npm install --save @svg-maps/portugal`
+
+### yarn
+
+`yarn add @svg-maps/portugal`
+
+## Credits
+
+This map is based on the work of [Lokal_Profil](https://commons.wikimedia.org/wiki/User:Lokal_Profil).
+
+The original map is available [here](https://commons.wikimedia.org/wiki/File:Australia_map,_States.svg) under the [Creative Commons Attribution-Share Alike 3.0 Unported](https://creativecommons.org/licenses/by-sa/3.0/deed.en) license.
```

Comment:
Could you please indicate the real credits and information about the original map?